testutil/fstest/file.go

```diff
@@ -56,6 +56,13 @@ func CreateDir(name string, perm os.FileMode) Applier {
 	})
 }
 
+// RemoveDir returns a file applier which removes the provided directory name
+func RemoveDir(name string) Applier {
```

Comment:
Maybe, it should reflect shallow `Remove` and `RemoveAll`. Those are familiar semantics.